include/lsst/sphgeom/CompoundRegion.h

```diff
@@ -68,6 +67,9 @@ class CompoundRegion : public Region {
     CompoundRegion &operator=(CompoundRegion const &) = delete;
     CompoundRegion &operator=(CompoundRegion &&) = delete;
 
+    // Return number of operands
+    unsigned int nOperands() const { return _operands.size(); }
```

Comment:
Unless there's precedent in sphgeom for using `unsigned int` for sizes of things, I think `std::size_t` would be better.